src/main/java/com/butbetter/applicationServices/CalculatorService.java

```diff
@@ -0,0 +1,13 @@
+package com.butbetter.applicationServices;
+
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.ResourceAccessException;
+
+public interface CalculatorService {
+
+    //public float calculateVATofProduct(Product product, float percent);
+
+    //public float calculateVATofProducts(List<Product> products, float percent);
+
+    float calculateVATofPrice(float price, float percent) throws ResourceAccessException, HttpClientErrorException.BadRequest;
```

Comment:
Which ResourceAccessException do you mean?

Reply:
spring boot throws a ResourceAccessException, if there is no answer from the http request.
https://stackoverflow.com/questions/45305811/throwing-resourceaccessexception-vs-httpclienterrorexception-for-resttemplate-cl